tsconfig.json

```diff
@@ -24,6 +24,6 @@
     "noFallthroughCasesInSwitch": true,
     "types": ["node"]
   },
-  "include": ["./src"],
+  "include": ["./src", "src/client/components/atoms/collapse"],
```

Comment:
whaaaaaaat? i believe this is not needed. typescript will acces everything under `src` folder